packages/multimedia/gmmlib/package.mk

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="gmmlib"
+PKG_VERSION="50707fb182bc8fc8f9711d2c0da96f6ded1b8ef4"
+PKG_SHA256="73c8895e3d5c54c8a44682c69db5c1a40da5500dacc07c89cb520d2b21626b05"
+PKG_ARCH="x86_64"
+PKG_LICENSE="MIT"
+PKG_SITE="https://01.org/linuxmedia"
+PKG_URL="https://github.com/intel/gmmlib/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_LONGDESC="gmmlib: The Intel(R) Graphics Memory Management Library provides device specific and buffer management for the Intel(R) Graphics Compute Runtime for OpenCL(TM) and the Intel(R) Media Driver for VAAPI."
+PKG_TOOLCHAIN="cmake"
```

Comment:
needed if only cmake is at the package ?

Reply:
removed